### PR TITLE
vGPU lane

### PR DESCRIPTION
**Pull request by vladikr**
This PR will add a new job to test the vGPU/mdev creation and allocation.

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 1]
```diff
@@ -915,6 +915,67 @@ presubmits:
...
+    cluster: prow-workloads
+    decorate: true
+    decoration_config:
+      grace_period: 30m0s
+      timeout: 4h0m0s
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-shared-images: "true"
+      rehearsal.allowed: "true"
```
Comment (fgimenez) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:931]:
```suggestion
      preset-bazel-cache: "true"
      preset-bazel-unnested: "true"
```

File: github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml [hunk 2]
```diff
@@ -915,6 +915,67 @@ presubmits:
           path: /dev/vfio/
           type: Directory
         name: vfio
+  - always_run: false
+    annotations:
```
Comment (fgimenez) [inline github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:919]:
```suggestion
    optional: true
    annotations:
```

Comment (vladikr) [reply github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml:919]: Done

**Review by fgimenez — APPROVED** [review]
/approve

Awesome, thanks!

**Pull request merged**